### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by cesidio**

### CODE REVIEW

File: app/form_models/school_form.rb [hunk 1]
```diff
@@ -1,10 +0,0 @@
-class SchoolForm < VacancyForm
```
Comment (david-mears-dfe) [inline app/form_models/school_form.rb:1]: We still need this form. https://github.com/DFE-Digital/teacher-vacancy-service/blob/master/app/controllers/hiring_staff/vacancies/schools_controller.rb

File: app/models/concerns/vacancy_job_location_validations.rb [hunk 2]
```diff
@@ -1,7 +0,0 @@
-module VacancyJobLocationValidations
-  extend ActiveSupport::Concern
-
-  included do
-    validates :job_location, presence: true
```
Comment (david-mears-dfe) [inline app/models/concerns/vacancy_job_location_validations.rb:5]: Why get rid of the presence validation?

File: lib/tasks/export_tables_as_csv_to_big_query.rake [hunk 3]
```diff
@@ -1,10 +0,0 @@
-namespace :tables_as_csv do
-  desc 'Exports postgres tables to Big Query'
-  namespace :to_big_query do
-    task export: :environment do
-      # It doesn't use CSV anymore, but I'm not changing the task name for now to avoid having to change more of the
-      # infrasctructure than strictly necessary
-      ExportTablesToBigQueryJob.perform_later
```
Comment (david-mears-dfe) [inline lib/tasks/export_tables_as_csv_to_big_query.rake:7]: I thought this was how Steven got his data?

File: lib/tasks/update_database_records_in_big_query.rake [hunk 4]
```diff
@@ -1,8 +0,0 @@
-namespace :database_records do
-  desc 'Updates Big Query user and approver tables with latest records from DSI'
-  namespace :in_big_query do
-    task update: :environment do
-      ExportDsiUsersToBigQueryJob.perform_later
```
Comment (david-mears-dfe) [inline lib/tasks/update_database_records_in_big_query.rake:5]: Again, I thought this was how Steven got his data?

**Review by david-mears-dfe — CHANGES_REQUESTED** [review]
Some decisions I disagree with, some I am confused by.

File: app/form_models/school_form.rb (hunk 1 above)
Comment (cpjmcquillan) [reply app/form_models/school_form.rb:1]: No we don't?

File: lib/tasks/export_tables_as_csv_to_big_query.rake (hunk 3 above)
Comment (cpjmcquillan) [reply lib/tasks/export_tables_as_csv_to_big_query.rake:7]: We don't use the rake tasks

File: app/models/concerns/vacancy_job_location_validations.rb (hunk 2 above)
Comment (cesidio) [reply app/models/concerns/vacancy_job_location_validations.rb:5]: I made a search for module VacancyJobLocationValidations, it is not included anywhere. In order to find the code that is not used, I have a look here:
https://sonarcloud.io/component_measures?id=DFE-Digital_teacher-vacancy-service&metric=coverage&view=list

Comment (cpjmcquillan) [reply app/models/concerns/vacancy_job_location_validations.rb:5]: We already do `validates :job_location, presence: true` on the form model

File: lib/tasks/export_tables_as_csv_to_big_query.rake (hunk 3 above)
Comment (cesidio) [reply lib/tasks/export_tables_as_csv_to_big_query.rake:7]: When we were on AWS, the way to do cron jobs was to create rake tasks and set up some kind of cronjobs that invoked them. When we moved to GOV.UK PaaS, we now have Sidekiq Cron, a gem that invokes directly the jobs, no need to create rake tasks. You could argue that when you have to run them manually, you can invoke them, but calling the job from within the console is not the same amount of typing.

File: lib/tasks/update_database_records_in_big_query.rake (hunk 4 above)
Comment (cesidio) [reply lib/tasks/update_database_records_in_big_query.rake:5]: same as before

**Review by cpjmcquillan — APPROVED** [review]

File: app/form_models/school_form.rb (hunk 1 above)
Comment (david-mears-dfe) [reply app/form_models/school_form.rb:1]: The link is to a file which uses it. This PR doesn't delete that file. So we are still using it. What am I missing?

Comment (cesidio) [reply app/form_models/school_form.rb:1]: it's `SchoolsForm` not `SchoolForm`

**Review by david-mears-dfe — APPROVED** [review]

**Pull request merged**